extensions/teams/infra/azure.json

```diff
@@ -1,220 +0,0 @@
-{
```

Comment:
Is this not required for the one click deployment to work?

Reply:
One click deploy for Teams has not been implemented. Provision, deploy, and publish are handled through the VSCode Teams Toolkit.